source/docs/articles/sites/migrate/wordpress-site-networks.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Migrate to Pantheon: WordPress Site Networks
+description: Learn how to import a WordPress Site Network into Pantheon.
+keywords: import, importing site, pantheon, new site, large site, distro, upstream, git history
+---
+
+## Requirements
+
+* [Download](http://git-scm.com/downloads) and install [Git](/docs/articles/local/starting-with-git/)
+* [Rsync or SFTP Client](https://pantheon.io/docs/articles/local/rsync-and-sftp/)
+* [MySQL Client](https://pantheon.io/docs/articles/local/accessing-mysql-databases/)
+* A Pantheon employee must create a [WordPress Site Network](/docs/articles/wordpress/site-networks/) for you.
+
+## Import the Codebase
+
+**Codebase** - all executable code, including core, custom and contrib modules or plugins, themes, and libraries.
+
+You can use Git to import your existing code and commit history. If you don’t have a Git version controlled codebase, the following will still work.
+
+1. Navigate to your existing site's code directory in a local terminal. If your existing code is not version controlled with Git, run:
+
+ ```bash
+ git init
+ ```
+2. From the Dev environment of the Site Dashboard, set the site's [connection mode](/docs/articles/getting-started/#interact-with-your-code) to Git.
+3. Copy the SSH URL for the site repository, found in the [clone command](/docs/articles/local/starting-with-git/#step-2-copy-the-git-clone-command). **Do not copy `git clone` or the site name.** The URL should look similar to the following:
+
+ ```bash
+ ssh://codeserver.dev.{site-id}@codeserver.dev.{site-id}.drush.in:2222/~/repository.git
+ ```
+
+4. Use Git to pull in the upstream's code (which may have Pantheon-specific optimizations) to your existing site's codebase, replacing `<ssh_url>` with the SSH URL copied in step 3:
+
+ ```bash
+ git pull --no-rebase --squash -Xtheirs <ssh_url> master
+ ```  
+
+ Will yield:  
+ ```bash
+ Squash commit -- not updating HEAD  
+ Automatic merge went well; stopped before committing as requested
+ ```

```

Comment:
@rachelwhitton this should be covered by requirements - link to "Export existing WordPress site"